pyqtgraph/functions.py

```diff
@@ -1179,10 +1179,9 @@ def imageToArray(img, copy=False, transpose=True):
             # If this works on all platforms, then there is no need to use np.asarray..
             arr = np.frombuffer(ptr, np.ubyte, img.byteCount())
     
+    arr = arr.reshape(img.height(), img.width(), 4)
```

Comment:
I was tricked by the 'RGB' part, but should have realized that it was still 4 bytes because the '32'. Durr. Thanks for the explanation :smile: